net/relayd/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		relayd
-PLUGIN_VERSION=		1.2
+PLUGIN_VERSION=		2.0
```

Comment:
If you call this 2.0.d we can reserve 2.0 for when it goes live and until then only bump PLUGIN_REVISION

Reply:
You mean 2.0.1, 2.0.2 ...?

Reply:
No:

PLUGIN_VERSION= 2.0.d
PLUGIN_REVISION=0

produces 2.0.d, then revision 1 is 2.0.d_1, etc.

On the actual release day that still leaves 2.0 as an update option.